diagnostic.md

```diff
@@ -184,5 +203,5 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+30px
 <!-- your answer ends here -->
```

Comment:
137.5 px - Reason being that since it's min-width, the media query will have .stretch be 25% and at 550px width, 25% is 137.5 px (which actually comes out to 137px).